packages/core/test/integration/data-types/data-types.test.ts

```diff
@@ -729,8 +750,10 @@ describe('DataTypes', () => {
       });
 
       it('rejects unsafe integers', async () => {
-        await expect(vars.User.create({ bigintAttr: 9_007_199_254_740_992 })).to.be.rejected;
-        await expect(vars.User.create({ bigintAttr: -9_007_199_254_740_992 })).to.be.rejected;
+        if (dialect.name !== 'oracle') {
```

Comment:
This should be rejected

Reply:
Support of `Bigint` was added in `oracledb 6.7`. I have bumped up the version. The test is updated to reject.